installation/resources/installer-local.yaml

```diff
@@ -21,7 +21,7 @@ spec:
     default:
       # If a container does not specify memory limit, this default value will be applied.
       # If a container tries to allocate more memory, container will be OOM killed.
-      memory: 96Mi
+      memory: 256Mi
```

Comment:
Note to reviewer: at 96Mi go-getter is failing immediately with "cannot allocate memory". At 128Mi it's failing in the middle of installation. At 256Mi it works with current release archive kyma-release-1.9.zip